src/python/pants/backend/awslambda/python/target_types.py

```diff
@@ -32,6 +32,17 @@
 
 class PythonAwsLambdaSources(PythonSources):
```

Comment:
Unlike `pex_binary`, we are guaranteed that the `handler` field is already defined. 

Also, they can still use the fully specified module path; but they will need to use the file name scheme for file args to work.